lib/rspec/mocks/minitest_integration.rb

```diff
@@ -0,0 +1,15 @@
+require 'rspec/mocks'
+
+Minitest::Expectation.class_eval do
+  def to(*args, &block)
+    ::RSpec::Mocks::ExpectationTarget.new(target).to(*args, &block)
+  end
+
+  def not_to(*args, &block)
+    ::RSpec::Mocks::ExpectationTarget.new(target).not_to(*args, &block)
+  end
+
+  def to_not(*args, &block)
+    ::RSpec::Mocks::ExpectationTarget.new(target).to_not(*args, &block)
+  end
+end
```

Comment:
BTW, once we do that I think the [cuke](https://github.com/rspec/rspec-mocks/blob/master/features/outside_rspec/minitest.feature) should be updated to use the new `rspec/mocks/minitest_integration` file instead of manually integrating it.  However, we still need a cuke showing how rspec-mocks can integration with _any_ test framework by calling the `setup`/`verify`/`teardown` methods at the appropriate time...so I think we should add a new cuke to take the place of the minitest one that integrates manually with some other test framework.  Maybe with the test-unit gem?  (As opposed to test-unit in stdlib which subclasses minitest in Ruby 1.9+).